packages/webpack-cli/lib/groups/ConfigGroup.js

```diff
@@ -4,14 +4,25 @@ const { extensions } = require('interpret');
 const GroupHelper = require('../utils/GroupHelper');
 const rechoir = require('rechoir');
 
+// Order defines the priority, in increasing order
 const DEFAULT_CONFIG_LOC = [
     '.webpack/webpack.config',
+    '.webpack/webpack.config.none',
     '.webpack/webpack.config.dev',
+    '.webpack/webpack.config.development',
     '.webpack/webpack.config.prod',
+    '.webpack/webpack.config.production',
     '.webpack/webpackfile',
+    'webpack.config.development',
     'webpack.config',
+    'webpack.config.production',
 ];
 
+const modeAlias = {
+    production: 'prod',
+    development: 'dev',
+};
```

Comment:
What about none?

Reply:
none has no alias, no need

Reply:
It matches either mode or it's alias, none will match the mode, no need alias

Reply:
@evilebottnawi

Reply:
Can you add comment how we prefer configurations here for future developers?